mxcube3/ui/reducers/taskForm.js

```diff
@@ -1,15 +1,16 @@
 const initialState = {
-  sample_ids: [],
+  sampleIds: [],
```

Comment:
Yes I try to use this everywhere but fail sometimes. Every time I refactor anything I will change it to camelcase, if you agree.